### PR TITLE
Change artefacts name

### PR DESCRIPTION
**Pull request by dincho**
https://www.pivotaltracker.com/story/show/163189858

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ references:
       - image: aeternity/infrastructure
     working_directory: ~/aeternity
     environment:
-      DOCKERHUB_REPO: aeternity/epoch
+      DOCKERHUB_REPO: aeternity/aeternity
```
Comment (lucafavatella) [inline .circleci/config.yml]: What is the plan for the following tags in aeternity/epoch?
* master
* minerva
* latest

I suggest:
* After this PR merged, delete `master` and `minerva` from aeternity/epoch.
* After aeternity/aeternity latest pushed as 1.3.0, delete latest from aeternity/epoch.

Comment (dincho) [reply .circleci/config.yml]: Good point I thought `latest` has been migrated as part of https://www.pivotaltracker.com/story/show/163298160

I'll migrate it before merging this.

Then once this is merged I'll delete master and minerva from aeternity/epoch and latest once 1.3.0 is tagged, unfortunately that will break the BC, maybe we push 1.3.0 and `latest` in `aeternity/epoch` as well to give some gratis period?

Comment (dincho) [reply .circleci/config.yml]: `latest` has been pushed to the new repo https://hub.docker.com/r/aeternity/aeternity/tags

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   node1:
     build: .
-    image: aeternity/epoch:${IMAGE_TAG-latest}
+    image: aeternity/aeternity:${IMAGE_TAG-latest}
```
[the remark this answers is not in the document]
Comment (dincho) [reply docker-compose.yml]: done: https://hub.docker.com/r/aeternity/aeternity/tags

File: scripts/install.sh [hunk 3]
```diff
@@ -84,12 +97,18 @@ install_node() {
     rm "${TEMP_RELEASE_FILE}"
 }
 
+# Backward compatibility package names
+# @TODO remove after 2.* release
+if in_array OLD_RELEASE_VERSIONS $RELEASE_VERSION; then
```
[the remark this answers is not in the document]
Comment (dincho) [reply scripts/install.sh]: me too, I found `in_array` function in the wild

**Comment by dincho** [comment]
> Worth a line in release notes, that canonical DockerHub location changed.

add in 60ed7ca5

**Pull request merged**